spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -232,7 +234,15 @@ def get_parser():
              "For more details on the subjects and methods used to create the models, go to: "
              "https://github.com/sct-pipeline/ukbiobank-spinalcord-csa#readme \n"  # TODO add ref of the paper
              "Given the risks and lack of consensus surrounding CSA normalization, we recommend thoroughly reviewing "
-             "the literature on this topic before applying this feature to your data.\n"
+             "the literature on this topic before applying this feature to your data.\n\n"
```

Comment:
Nit: There's an extra `\n` from the old description.

Reply:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/9e06ffcab367ca361319f3ec584bbd19fe23c2a9